Northstar.Custom/mod/scripts/vscripts/sh_northstar_http_requests.gnut

```diff
@@ -89,6 +89,7 @@ void function NSHandleSuccessfulHttpRequest( int handle, int statusCode, string
 		response.statusCode = statusCode
 		response.body = body
 		response.rawHeaders = headers
+		response.headers = {}
```

Comment:
Unrelated change?

Reply:
AFAIK @Erlite has become inactive on this account. Maybe @Zanieon could take this PR over?

Reply:
I haven't worked with anything on the HTTP requests of Northstar yet, judging by those functions code i'm uncertain how their logic functions, especially if there's integration at Launcher side.